modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/AppDatabase.kt

```diff
@@ -989,6 +997,45 @@ abstract class AppDatabase : RoomDatabase() {
             database.execSQL("ALTER TABLE podcasts ADD COLUMN funding_url TEXT")
         }
 
+        val MIGRATION_113_114 = addMigration(113, 114) { database ->
+            with(database) {
+                beginTransaction()
+                try {
+                    database.execSQL(
+                        """
+                        CREATE TABLE IF NOT EXISTS notification(
```

Comment:
```suggestion
                        CREATE TABLE IF NOT EXISTS notifications(
```
Should the table name be a plural? We have done this for others such as `podcasts` and `folders`.